cico_build_ci.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Output command before executing
+set -x
+
+# Exit on error
+set -e
+
+SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
+export SCRIPT_DIR
+
+# shellcheck disable=SC1090
```

Comment:
Instead of disabling the shellcheck directive, you can instead specify
```
# shellcheck source=cico_functions.sh
```

Reply:
fixed